Drivers/seds/PioHwas/example/example.asn

```diff
@@ -0,0 +1,42 @@
+TEST3-DATAVIEW DEFINITIONS ::=
+BEGIN
+
+   MyInteger ::= INTEGER (0 .. 10000)
+
+-- ASN.1 Type definitions must start with an uppercase
```

Comment:
The unused/commented out ASN.1/ACN definitions could be removed to avoid clutter

Reply:
I guess I could completely remove this file and example.acn. 
@Lurkerpas

Reply:
I am not sure if removing this file entirely won't break the project. You could try.

Reply:
Fixed. Files deleted @Lurkerpas